src/osd/ReplicatedBackend.cc

```diff
@@ -1988,13 +1988,15 @@ int ReplicatedBackend::build_push_op(const ObjectRecoveryInfo &recovery_info,
     for (iter->lower_bound(progress.omap_recovered_to);
 	 iter->valid();
 	 iter->next(false)) {
+      uint64_t size = iter->key().size() + iter->value().length();
```

Comment:
This is an unnecessary refactor, I don't think it should be included in this branch.